js/build.sh

```diff
@@ -8,8 +8,19 @@ wasm-pack build \
     --out-dir $2 \
     --release
 
+# Add 'crypto' polyfill to js libs
+echo "
+// Polyfill to enable signing in some JS environments
+// See: https://stackoverflow.com/questions/52612122/how-to-use-jest-to-test-functions-using-crypto-or-window-mscrypto
+const crypto = require('crypto');
+Object.defineProperty(global.self, 'crypto', {
+  value: {
+    getRandomValues: arr => crypto.randomBytes(arr.length)
+  }
+});
+" >> $2/doughnut.js
```

Comment:
Do we need to add that to doughnut.js? I think it's only necessary for jest and thus we should only add it to our doughnut.spec.js

Reply:
we want to export it as part of the lib by default so it can run anywhere.
It could fail in browser for instance without this